law_reader/summarize_etl.py

```diff
@@ -0,0 +1,98 @@
+from supabase import create_client, Client
+import os
+
+from common.RevisionSummaryInfo import RevisionSummaryInfo
+from summarizer.SummarizationException import SummarizationException
+from summarizer.summarization import Summarization
+
+"""
+File for downloading full text of bills from Supabase, summarizing them, and uploading the summaries to Supabase
+
+Pulls data from:
+- Revisions
+- Revision_Text
+
+The database tables updated by this script are:
+- Summaries
+"""
+
+DEBUG = True  # Set to True to print debug messages
+if DEBUG:
+    from dotenv import load_dotenv
+    load_dotenv()  # Load environment variables from .env file
```

Comment:
In Github Actions, there's no .env file -- just Github secrets which are automatically loaded into the environment. By contrast, when using local development, load_dotenv() allows us to put things from .env into that environment. 

I'll work this inside a main guard. Additionally, since this patch of code exists a few times, I'll look into whether I can make it more DRY.